src/parser.js

```diff
@@ -401,86 +425,118 @@ export default class Parser {
 
       // Transactions are created for each response in the document
       _.each(relevantResponses, (responseValue, statusCode) => {
-        let examples = {
-          '': undefined,
-        };
+        this.handleSwaggerResponse(resource, transition, method,
+          methodValueParameters, responseValue, statusCode);
+      });
 
-        this.path.push('responses');
-        this.path.push(statusCode);
+      return transition;
+    });
+  }
 
-        if (responseValue.examples) {
-          examples = responseValue.examples;
-        }
+  // Convert a Swagger response & status code into Refract transactions.
+  handleSwaggerResponse(resource, transition, method, transitionParameters, responseValue, statusCode) {
+    let examples = {
+      '': undefined,
```

Comment:
I'll add a comment to the source, but the gist is that the Refract spec *requires* an HTTP Transaction to have a request and a response, so this seeds the examples list to create a blank request/response. Why not just leave out the transaction, you ask? That is the only way we have of specifying which HTTP methods are allowed, so we need to create a transaction.